tests/AppTest.php

```diff
@@ -564,7 +564,7 @@ public function testMultipleAliases(string $path, string $expected): void
         $result = \ob_get_contents();
         \ob_end_clean();
 
-        var_dump($result);
+        // var_dump($result);
```

Comment:
We can remove this comment.

Reply:
Removed var_dump alltogether. Also from 3 other places in this test (they were commented out)